contrib/garyburd/redigo/redigo.go

```diff
@@ -121,6 +121,10 @@ func (tc Conn) Do(commandName string, args ...interface{}) (reply interface{}, e
 		}
 	}
 
+	if ctx == nil {
+		ctx = context.Background()
+	}
```

Comment:
These were added to make go 1.15 happy. It does not accept nil context anymore when starting a new context and redis library/instrumentation was guilty of passing around nil context.